components/list/list-item-mixin.js

```diff
@@ -18,6 +18,9 @@ import { ListItemRoleMixin } from './list-item-role-mixin.js';
 import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
 import ResizeObserver from 'resize-observer-polyfill';
 import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
+import { styleMap } from 'lit-html/directives/style-map.js';
+
+const HEX_REGEX = /#([0-9a-fA-F]){6}(?:[0-9a-fA-f]{2})?/; // 6 or 8 character hex code
```

Comment:
We could create a `color.js` helper that contains a method `isHexColor(alpha)`. This Regex could be moved there, as well as the one that `input-color.js` uses. We could have a few unit tests to cover these in isolation from the components.

Related: when I did my inFusion Dark Mode work, I think I ended up creating a similar color.js that had a lot of color functions, including color parsing, validation, and other things. I see that we have a `contrast.js` helper that I think was a contribution from outside Gaudi - it's functionality could also be represented in `color.js` but I would want to refactor it, and it's probably used by someone outside of core.